projects/packages/my-jetpack/_inc/hooks/use-fullscreen/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+
+/**
+ * Goes into fullscreen mode when the component is loaded
+ *
+ */
+export const useFullScreen = () => {
+	useEffect( () => {
+		const hasToolbarClass = document.documentElement.classList.contains( 'wp-toolbar' );
+
+		document.body.classList.add( 'jetpack-admin-full-screen' );
+		document.body.classList.add( 'is-wp-toolbar-disabled' );
```

Comment:
Why is it necessary to have two classes added to the same HTML element?

In my understanding, everything could be under `jetpack-admin-full-screen`. The logic of this hook also doesn't suggest otherwise. 🤔